HOWTO_meta_quest.md

```diff
@@ -235,71 +256,60 @@ More capture options can be found in the
 [USAGE_android.md](./USAGE_android.md) under the [Capture
 Options](./USAGE_android.md#capture-options) section.
 
-### 4. Install and Run the Application
-
-  1. Open Android Studio 
-  2. Connect the headset to your development
-
-    machine using a USB plug
-    a. Make sure the headset is detected by Android Studio
-  3. Select the headset in the target 
-  4. Make sure "app-openxr-vulkan"
-
-    is selected 
-  5. Press the "Play" triangle to run
+### 7. Install and Run the Application
 
-    a. You may have to first select to enable commands from the host
-    system on the headset
+Install the application as normal and run it.
+You may have to stop and uninstall any older 
 
-This shows that you can run the application on the target.
+Then run the application as normal to capture
+the content.
 
-### 5. Run the Application
-
-Repeat the steps from section 2. above.
-
-This time, however, you should see a generated capture file after
-exiting the application in the `/sdcard/Download` folder called
-`openxr_capture_frames_1_to_500.gfxr` since you have selected to save
-only some of the frames.
-
-### 6. Verify the capture file
-
-Check that the above file exists:
+If the application fails to launch, review the
+`adb logcat` information for any fatal or error messages.
+Here is a useful filter for logcat messages.
 
 ```bash
-adb shell ls /sdcard/Download/openxr_capture*.gfxr
+adb logcat -s DEBUG vulkan VulkanLoader VulkanLoaderAndroid OpenXR OpenXR-Loader gfxrecon
 ```
 
-### 7. Disable the capture layer
+Once complete, if everything executed correctly, the capture
+file should be present in the `/sdcard/Download` folder.
 
-Disable the capture layer globally and also restore the Vulkan usage of
-the HWUI:
+Check that it exists:
 
 ```bash
-adb shell "setprop debug.vulkan.layers ''"
-adb shell "settings put global gpu_debug_app ''"
-adb shell "settings put global gpu_debug_layers ''"
+adb shell ls /sdcard/Download/openxr_capture*.gfxr
```

Comment:
Do we want to point out the impact of git bash rewriting /sdcard into some other text and the solutions?